proto/ibc/applications/middleware/fee/v1/genesis.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// GenesisState defines the fee middleware genesis state
+message GenesisState {
+  // A mapping of packets -> escrowed fees
+  repeated ibc.applications.middleware.fee.v1.IdentifiedPacketFee packets_fees_map = 1
```

Comment:
I'd remove the `map` part since it is just an array of identified packets. `IdentifiedPacketFees` would be most consistent with what we already have